modules/ossm-kiali-ossmc-plugin-user-guide.adoc

```diff
@@ -0,0 +1,54 @@
+////
+This CONCEPT module included in the following assemblies:
+-service_mesh/v2x/ossm-kiali-ossmc.adoc
+////
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-kiali-ossmc-user-guide_{context}"]
+= About the OpenShift Service Mesh Console plugin
+//In the title include nouns or noun phrases that are used in the body text.
+//Do not start the title of concept modules with a verb..
+
+
+// Working on converting this to Asciidoc
+// Commenting out so there is delineation in this file 
+// DESCRIPTION LIST in asciidoc
+// -- are an open  block, help keep things together esp with bullets in the middle of stuff
+// plugin compatible with OCP 4.15+
+
+The OpenShift Service Mesh Console (OSSMC) plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh from the OpenShift Console. You can install the OSSMC plugin in only one of two ways: using the OpenShift Console, or through the CLI.
+
+Installing the OSSMC plugin creates a new category, *Service Mesh*, in the main OpenShift Console navigation. Click on *Service Mesh* to see:
+
+* *Overview* for a summary of your mesh displayed as cards that represent the namespaces in the mesh. 
+* *Graph* for a full topology view of your mesh represented by nodes and edges - each node representing a component of the mesh and each edge representing traffic flowing through the mesh between components.
+* *Istio config* for a list of all Istio configuration files in your mesh with a column that provides a quick way to know if the configuration for each resource is valid.
+
+
+Under *Workloads*, the OSSMC plugin adds a *Service Mesh* tab that contains the following subtabs:
+
+* *Overview* subtab provides a summary of the selected workload including a localized topology graph showing the workload with all inbound and outbound edges and nodes.
+* *Traffic* subtab displays information about all inbound and outbound traffic to the workload.
+* *Logs* subtab shows the logs for the workload's containers. 
++
+--
+** You can view container logs individually or in a unified fashion, ordered by log time. This is especially helpful to see how the Envoy sidecar proxy logs relate to your workload's application logs. Y
+** You can enable the tracing span integration which then allows you to see which logs correspond to trace spans.
+--
++
+* *Metrics* subtab shows both inbound and outbound metric graphs in the corresponding sub-tabs. All the workload metrics can be displayed here, providing you with a detail view of the performance of your workload. 
```

Comment:
Everywhere else "subtab" is one word - here it is hyphenated, which I assume is a typo.
```suggestion
* *Metrics* subtab shows both inbound and outbound metric graphs in the corresponding subtabs. All the workload metrics can be displayed here, providing you with a detail view of the performance of your workload. 
```